src/hooks/useVenueAccessToken.ts

```diff
@@ -0,0 +1,58 @@
+import { useEffect } from "react";
+import firebase from "firebase/app";
+
+import { AnyVenue } from "types/Firestore";
+
+import { WithId } from "utils/id";
+import { accessTokenKey } from "utils/localStorage";
+import { isTruthy, notEmpty } from "utils/types";
+import { checkAccess, getLocalStorageToken } from "functions/auth";
+
+export const useVenueAccessToken = (
```

Comment:
Maybe just call this `useVenueAccess`/`useVenueAccessCheck`, as `useVenueAccessToken` implies that it's just going to return me the token, which it doesn't seem to be doing.

```suggestion
export const useVenueAccess = (
```
```suggestion
export const useVenueAccessCheck = (
```